stonesoup/sensor/radar/radar.py

```diff
@@ -189,6 +189,7 @@ class RadarElevationBearingRange(RadarBearingRange):
         doc="The sensor noise covariance matrix. This is utilised by "
             "(and follow in format) the underlying "
             ":class:`~.CartesianToElevationBearingRange` model")
+    max_range: float = Property(doc="The maximum detection range of the radar (in meters)")
```

Comment:
`default=np.inf`?

Reply:
Yes I was thinking that, but wanted to be consistent with _RadarRotatingBearingRange_, maybe both need a change?